docs/source/deployment/cluster.rst

```diff
@@ -45,15 +45,13 @@ development machine and the cluster. Such a connection will simplify the file tr
 the user cluster password from being requested multiple times.
 
 .. attention::
-  The workflow has been tested with ``apptainer version 1.2.5-1.el7`` and ``docker version 24.0.7``.
+  The workflow has been tested with:
 
-  - ``apptainer``:
-    There have been reported binding issues with previous versions (such as ``apptainer version 1.1.3-1.el7``). Please
-    ensure that you are using the latest version.
-  - ``Docker``:
-    The latest versions (``25.x``) cannot be used as they are not compatible yet with apptainer/ singularity.
+  - ``apptainer version 1.2.5-1.el7`` and ``docker version 24.0.7``
+  - ``apptainer version 1.3.4`` and ``docker version 27.3.1``
+
+  In the case of issues, please try to switch to those versions.
 
-    We are waiting for an update from the apptainer team. To track this issue, please check the `forum post`_.
```

Comment:
Can you remove the link to the forum post indicated here as well? :)

Reply:
done